01.general/10.why-is-adguard-better/docs.zh_CN.md

```diff
@@ -0,0 +1,47 @@
+---
+title: '为什么 AdGuard 优于其它广告拦截器'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+AdGuard 常与熟知的浏览器扩展器对比 - 即 Adblock 和 Adblock Plus。我们利用本文描述 AdGuard 具备但其对手所不具备的主要优势。
+
+Adguard 的最大优势在于其不受限于浏览器的能力限制。Adblock 和 Adblock Plus 从技术上讲是无法移除网页上的所有广告的。AdGuard 是在广告元素加载到浏览器之前就先拦截了它们。而扩展的工作是基于浏览器的广告拦截能力之上。这就是为什么在隐藏拦截的元素之后需要进行 [后处理](http://adguard.com/en/how-ads-blocked.html#cosmetic) 。如您曾经使用过 Adblock，则您肯定知道隐藏后的横幅广告是如何 “闪的”。
+
+![](banner.png)
```

Comment:
The screenshot link is： <img src="https://kb.adguard.com/user/pages/01.general/10.why-is-adguard-better/banner.png" />